Gemfile

```diff
@@ -91,13 +91,15 @@ end
 group :test do
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '>= 2.15', '< 4.0'
+  gem "poltergeist"
   gem 'capybara-webkit', require: false
   gem 'cucumber-rails',  require: false
   # Easy installation and use of chromedriver to run system tests with Chrome
   gem 'chromedriver-helper'
   gem 'database_cleaner'
   gem 'factory_bot_rails'
   gem 'guard-rspec', require: false
+  gem "phantomjs", require: 'phantomjs/poltergeist'
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.